### PR TITLE
docs(drag-drop): add docs for cdkDragFreeDragPosition

### PR DESCRIPTION
**Pull request by crisbeto**
Adds docs and a live example of the `cdkDragFreeDragPosition` input.

### CODE REVIEW

File: src/cdk/drag-drop/drag-drop.md [hunk 1]
```diff
@@ -199,3 +199,11 @@ will wait for the user to hold down their pointer for the specified number of mi
 moving the element.
 
 <!-- example(cdk-drag-drop-delay) -->
+
+### Changing the standalone drag position
+By default the only way to change the position of a standalone `cdkDrag` is for the user to move it
+around manually, however there are some cases where it's useful to set the element's position for
+the user (e.g. when restoring the state after the user has navigated and come back). In these cases
+you can tell `cdkDrag` where it should be positioned through the `cdkDragFreeDragPosition` input.
```
Comment (jelbourn) [inline src/cdk/drag-drop/drag-drop.md]: I would rephrase slightly for more active voice
```markdown
By default, standalone `cdkDrag` elements move from their normal DOM position
only when manually moved by a user. The element's position can be explicitly
set, however, via the `cdkDragFreeDragPosition` input. Applications
commonly use this, for example, to restore a draggable's position after a user
has navigated away and then returned.
```

Comment (crisbeto) [reply src/cdk/drag-drop/drag-drop.md]: Sounds much better. Changed.
